libs/image/website/src/lib/image.tsx

```diff
@@ -24,10 +24,11 @@ export const imageToSquareImageItems = (image: FullImageFragment): ImageItem[] =
   {url: image.squareSmallURL, size: 200}
 ]
 
-export const ImageWrapper = styled('img')`
+export const ImageWrapper = styled('img')<{aspectRatio: number}>`
   max-width: 100%;
   width: 100%;
   height: auto;
+  aspect-ratio: auto ${({aspectRatio}) => aspectRatio};
```

Comment:
Fixes image going haywire when the image can't load. As aspect-ratio by the user agent is only set when the image is loaded.

Reply:
shouldn't there be a default?

Reply:
This is the default